android/src/main/java/com/gettipsi/stripe/util/Converters.java

```diff
@@ -180,6 +180,28 @@ public static String getValue(final ReadableMap map, final String key) {
     return allowedCountriesForShipping;
   }
 
+  public static BankAccountTokenParams.Type accountHolderTypeFromString(String value) {
+    if (value == "individual") {
```

Comment:
In my case, this method always return null. So, I have to change from `value == "individual"` to `value.equals("individual")`

Reply:
Oh thank you for your review! 🙌  My Java is obviously pretty rusty, I'll try to push a commit with an `.equals` check as soon as possible!

Reply:
I pushed those changes you suggested, but wasn't able to test in the app I'm using this in myself because It doesn't build right now because of other reasons, and I'm on vacation for about a month so I don't have time to figure those details out right now. I trust that what you suggested is correct, but next time I'll be able to test to see that this is working myself is probably in the end of august, just an FYI.